package.json

```diff
@@ -7,15 +7,14 @@
   "scripts": {
     "build:css": "bin/build-css",
     "build": "bin/build",
-    "lint": "standard",
-    "unit-test": "lab -c -r console -o stdout -r lcov -o lcov.info",
     "test": "npm run lint && npm run unit-test",
     "start": "node index.js",
     "start-quiet": "node index.js >/dev/null",
+    "lint": "standard",
     "pa11y-test": "pa11y-ci",
     "snyk-test": "snyk test",
-    "ci-test": "start-server-and-test start-quiet 3000 pa11y-test",
-    "all-tests": "npm run test && npm run snyk-test && npm run ci-test"
+    "unit-test": "lab -c -r console -o stdout -r junit -o ./test-output/TEST-test.xml",
```

Comment:
A change I made on the other branch was to add a unique date/time stamp to the test output.

TEST-$(date '+%Y%m%d-%H%M%S').xml

Reply:
Good idea, I've copied that change in.